File: C/1036.c
#include <stdio.h>
#include <math.h>

double main(){
	double A, B, C, delta, resultado1, resultado2;
	scanf("%lf%lf%lf", &A, &B, &C);
	delta = pow(B,2)-4*A*C;
	if (delta<0 || 2*A==0){
		printf("Impossivel calcular\n");
	}
	else{
		resultado1 = (-B+(sqrt(delta)))/(2*A);
		resultado2 = (-B-(sqrt(delta)))/(2*A);
		printf("R1 = %.5lf\n", resultado1);
		printf("R2 = %.5lf\n", resultado2);
	}
	return 0;
}

File: Python/1036.py
a, b, c = [float(x) for x in input().split()]
delta = ((b**2)-4*a*c)
if(delta>=0 and (2*a)!=0):
    delta = delta ** (1/2)
    r1 = (-b + delta)/(2*a)
    r2 = (-b - delta)/(2*a)
    print("R1 = {:.5f}".format(r1))
    print("R2 = {:.5f}".format(r2))
else:
    print("Impossivel calcular")